projects/driver_controls/inc/driver_io.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+// Digital input pin definitions
+
+#define DRIVER_IO_POWER_SWITCH_PORT 2
```

Comment:
Port enums please

If you're only ever using them for initialization, you could just define port/pin together as the struct.